python/TestHarness/testers/RunApp.py

```diff
@@ -296,7 +301,12 @@ def testExitCodes(self, moose_dir, options, output):
             # We won't pay attention to the ERROR strings if EXPECT_ERR is set (from the derived class)
             # since a message to standard error might actually be a real error.  This case should be handled
             # in the derived class.
-            if options.valgrind_mode == '' and not specs.isValid('expect_err') and len( [x for x in filter( lambda x: x in output, specs['errors'] )] ) > 0:
+            if self.exit_code == 77:
+                self.setStatus(self.skip, "ABORT")
```

Comment:
Why `ABORT`? Or is that not even used anywhere, it just shows as skip?

Reply:
I wanted a way to distinguish a test harness initiated static skip and a dynamic skip that runs the app

Reply:
Maybe `CAPABILITY` or something? `ABORT` is pretty general